app.rb

```diff
@@ -0,0 +1,39 @@
+require 'sinatra/base'
+require_relative './lib/peep'
+require_relative './lib/user'
+
+class Chitter < Sinatra::Base
+    enable :sessions
+
+    get '/' do
+        erb :'peeps/index'
```

Comment:
Did you create a peeps folder with an index, list and new .erb file within it? Good organisation of files